testing/macros/src/main/scala/org/macros/Macros.scala

```diff
@@ -0,0 +1,16 @@
+package org.macros
+
+import scala.language.experimental.macros
+import scala.reflect.macros.blackbox.Context
+
+/**
+ * Created by sugakandrey.
+ */
+object Macros {
```

Comment:
what is this file for? Last comment, honest :smile: Remove this file and then somebody (maybe me will merge)